src/graphiql-fetcher.ts

```diff
@@ -0,0 +1,32 @@
+import { SubscriptionClient } from './client';
+
+export const graphQLFetcher = (subscriptionsClient: SubscriptionClient, fallbackFetcher: Function) => {
+  let activeSubscriptionId: number | null = null;
+
+  return (graphQLParams: any) => {
+    if (subscriptionsClient && activeSubscriptionId !== null) {
+      subscriptionsClient.unsubscribe(activeSubscriptionId);
+    }
+
+    if (subscriptionsClient && graphQLParams.query.startsWith('subscription')) {
```

Comment:
GraphiQL already comes with graphql. Is there a way to get it from that bundle? If not, I would suggest this:

1. if `operationName` is set, look for a match for `subscription <operationName>`.
2. if `operationName` is not set, check if subscription is present somewhere before the first `{` in the string.

I think we should only send it over the subscription transport if we're very confident that a subscription should happen, because the error messages from subscriptions aren't as easy to understand as the ones that are returned from the normal graphql executor.